src/app/[lng]/page.tsx

```diff
@@ -1,5 +1,24 @@
 import App from '../ui/app/App';
+import { WidgetContainer, Widgets } from '@/components/Widgets';
+import { Widget } from '@/components/Widgets/Widget';
+import { getCookies } from '@/app/lib/getCookies';
+import { WidgetSkeleton } from '@/components/Widgets/WidgetSkeleton';
 
 export default function Page() {
-  return <App starterVariant="default" />;
+  const variant = 'default'; // exchange
```

Comment:
We have this TabsMap.Exchange.variant. Wdyt?

Reply:
https://github.com/lifinance/jumper.exchange/pull/1023#discussion_r1639450829